ui/src/store/catalog.ts

```diff
@@ -2,9 +2,7 @@ import { Instance, types } from 'mobx-state-tree';
 import { Icons } from '../common/icons';
 
 const icons: { [catalog: string]: Icons } = {
-  official: Icons.Cat,
-  verified: Icons.Certificate,
-  community: Icons.User
+  catalog: Icons.Catalog
```

Comment:
shouldn't we mark this as `community` instead of `catalog` since we are getting `community` from the API call?

Reply:
And also are we using `community` anywhere anymore ??

Reply:
now there is only one icon for any type of catalog so kept it as `catalog`

Reply:
I guess as of now we are adding new catalogs as community catalog right ?

Reply:
yeah, I will update it